### PR TITLE
ILIKE operator support

### PR DESCRIPTION
**Pull request by AMAN2202**
https://github.com/questdb/questdb/issues/2585

### CODE REVIEW

File: core/src/main/java/io/questdb/griffin/engine/functions/regex/ILikeStrFunctionFactory.java [hunk 1]
```diff
@@ -24,9 +24,23 @@
...
+
 public class ILikeStrFunctionFactory extends AbstractLikeStrFunctionFactory {
     @Override
     public String getSignature() {
         return "ilike(SS)";
     }
+
+    @Override
+    public Function newInstance(int position, ObjList<Function> args, IntList argPositions, CairoConfiguration configuration, SqlExecutionContext sqlExecutionContext) throws SqlException {
+        return getLikeStrFunction( args, argPositions, Pattern.CASE_INSENSITIVE);
```
Comment (ideoma) [inline core/src/main/java/io/questdb/griffin/engine/functions/regex/ILikeStrFunctionFactory.java:44]: Should be both flags
```java
Pattern.DOTALL | Pattern.CASE_INSENSITIVE
```

File: core/src/main/java/io/questdb/griffin/engine/functions/regex/AbstractLikeStrFunctionFactory.java [hunk 2]
```diff
@@ -72,25 +72,39 @@ else if ("[](){}.*+?$^|#\\".indexOf(c) != -1) {
 
     @Override
     public Function newInstance(int position, ObjList<Function> args, IntList argPositions, CairoConfiguration configuration, SqlExecutionContext sqlExecutionContext) throws SqlException {
```
Comment (ideoma) [inline core/src/main/java/io/questdb/griffin/engine/functions/regex/AbstractLikeStrFunctionFactory.java:74]: not used, overriden, can be removed here

**Comment by pswu11** [comment]
Hi @AMAN2202 thanks for the pull request! This is a duplication to https://github.com/questdb/questdb/pull/2593
Since your solution came around the same time as the merged PR, we decided to label your PR as `hacktoberfest-accepted` so your effort can be recognized.

**Comment by puzpuzpuz** [comment]
Closing this one as a duplicate.

**Pull request closed without merging**